### ISSUE TITLE
Add `insertion` property for custom scripts

### ISSUE THREAD

**Issue by Clarko** [IvanMathy/Boop#75]
Scripts may want to add data at the current insertion point, rather than replace data.

Reason for request: I just wrote a Lorem Ipsum script and realized it can't just insert a lipsum paragraph into an existing string. It can only replace the `fullText` or replace a (non-zero-length) `selection`.

Totally invented use cases for insertion:

- get today's date
- roll a die
- flip a coin

**Comment by IvanMathy** [comment]
Interesting, this is a good idea. I feel like it could work nicely as a function maybe? Something like

```javascript
state.insert("Something")
```
In the meantime, you can probably get the same behavior with:

```javascript
state.text = state.text + "Something"
```

Not ideal, obviously.

**Comment by IvanMathy** [comment]
Fixed in #109, look for it in release 1.2.0!
